server/events/commit_status_updater.go

```diff
@@ -56,24 +55,23 @@ func (d *DefaultCommitStatusUpdater) UpdateCombined(repo models.Repo, pull model
 		descripWords = "succeeded."
 	}
 	descrip := fmt.Sprintf("%s %s", strings.Title(command.String()), descripWords)
-	return d.Client.UpdateStatus(repo, pull, status, src, descrip, "")
+	return d.Client.UpdateStatus(repo, pull, status, command.String(), descrip, "")
```

Comment:
This will change the form of the src argument from `atlantis/<command>` to just the command name. This src field  is used by most of the VCS plugins to drive a data field that is sent along in the PR status update requests. Changing the format of this field is going to be a breaking change to anyone with a PR policy keyed off of this particular string.

Reply:
We've been fighting #1172 as well and it's gotten to a point where we've had to disable the status policies altogether and rely on reviewers only. I'd like to 2nd what @acastle said, if I can help in any way to getting this merged let me know.

Would it make sense to use a different string just for DevOps users? It's already broken for them anyway and prevents breaking changes to other VCS users.

Reply:
@Jack64 I have submitted a separate PR #1209 that resolves just the status checks which was merged last week. There hasn't been a new release cut yet but if you do a new build off of master you should be all set.